libbeat/docs/processors-using.asciidoc

```diff
@@ -1146,6 +1147,38 @@ section.
 NOTE: If you define an empty list of fields under `include_fields`, then only
 the required fields, `@timestamp` and `type`, are exported.
 
+[[processor-registered-domain]]
+=== Registered Domain
+
+The `registered_domain` processor reads a field containing a hostname and then
+writes the "registered domain" contained in the hostname to the target field.
+For example, given `www.google.co.uk` the processor would output `google.co.uk`.
+In other words the "registered domain" is the effective top-level domain
+(`co.uk`) plus one level (`google`).
```

Comment:
Do you want to mention that this is calculated based on Mozilla's Public Suffix List, like we do in Packetbeat and ECS?

Reply:
Added